docs/content/operations/use_sbt_to_build_fat_jar.md

```diff
@@ -0,0 +1,102 @@
+libraryDependencies ++= Seq(
```

Comment:
would you mind adding the yaml front-matter and format the sbt code in a code block? That would make it friendlier to use.